### PR TITLE
Flash tests: Also set the nodes' min_fee_pct to 0

### PR DESCRIPTION
**Pull request by Geod24**
Also comes with two improvements to the documentation / messages we get.
@omerfirmak Please review.

### CODE REVIEW

File: source/agora/node/Config.d [hunk 1]
```diff
@@ -228,8 +228,31 @@ public struct NodeConfig
...
+        fee is at least `min_fee_pct` percent of the current transaction pool.
+        This setting can be tuned to prevent DoS attack.
+
+        Setting this to 0 sets the mempool to be unlimited in size.
+
+        Example:
+        The transaction pool has two transactions, `TxA` which is 1000 bytes and
+        has a fee of 15_000_000 (fee rate: 15_000) and another one which is 200
+        bytes and comes with a fee of 1_000_000 (fee rate: 5_000).
+        Hence, the average fee rate of the transaction pool is 13_333 / byte.
```
Comment (omerfirmak) [inline source/agora/node/Config.d]:
```suggestion
        Hence, the average fee rate of the transaction pool is 10_000 / byte.
```

Our average is not weighted. I wonder if it should be.

Comment (Geod24) [reply source/agora/node/Config.d]: That's what I suspected. Couldn't someone then rather easily lower one's requirement by sending a stream of txs which are just at the threshold, provided the pool's threshold is < 100 ?

Comment (omerfirmak) [reply source/agora/node/Config.d]: Yeah, they could. But, once the network is mature enough, they would need a lot UTXOs to be able to make any significant impact.

**Comment by Geod24** [comment]
Fixed

**Comment by Geod24** [comment]
Self merging because it gets the CI back on tracks.

**Pull request merged**